theme.css

```diff
@@ -82,12 +82,42 @@
   --bg-color: #ffffff;
   --rdp-bg-color: #f0f0f0;
   --rdp-text-color: #000000;
-  --rdp-cw-color: #9E9E9E;
+  --rdp-cw-color: #9e9e9e;
   --rdp-link-color: #8d8d8d;
   --rdp-details-color: #5b5b5b;
   --rdp-border-color: #cccccc;
 }
 
+:root[data-theme="deuteranomaly"] img,
```

Comment:
Inaccurate media element filters with `hue-rotate()` and `contrast()`:
![color-blind_media_filters](https://github.com/hyphacoop/reader.distributed.press/assets/68826419/8cf47c6d-f3b1-42e5-bf7a-b9028b46468c)